### ISSUE TITLE
XQuery 3.1 Map Syntax

### ISSUE THREAD

**Issue by jmeischner** [wcandillon/xqlint#77]
It seems that xqlint does not support the map syntax of XQuery 3.1.

```sh
19 |function _:schema-render-table-prepare-rows-jf($Items as element()*, $Schema as element(schema),$Context as map(*))
	                                                                                                                   ^ lexical analysis failed
while expecting '(:'
after successfully scanning 1 characters beginning at line 19, column 112:
...(*))
```

Is it possible to add this feature to xqlint?

**Comment by joewiz** [comment]
This feature was added in @wolfgangmm's fork for eXist: https://github.com/wolfgangmm/xqlint/commit/2b0479d9007b32de173194c8a5c39fad50a5f483.

**Comment by wcandillon** [comment]
@joewiz is there a feature you would like to see merge in the main package? Let me know.

**Comment by jmeischner** [comment]
Thank you for your answer. Is there a reason that this feature is not merged back into this repo? 

I created the [sublime linter package](https://github.com/jmeischner/SublimeLinter-contrib-xqlint) for this linter and it's much easier to install it simply by using npm than it seem to install the forked one.

**Comment by jmeischner** [comment]
Is there anything I can do to push this topic?

**Comment by joewiz** [comment]
@wcandillon I would certainly support the idea of merging this feature into the main xqlint. 

@jmeischner I'm sure @wolfgangmm would be happy to see the feature pulled into the main xqlint project. Would you be able to prepare a PR?

**Comment by wcandillon** [comment]
Yes please guys go for it! Definitely would love to see it in the main xqlint. I even had a proof of concept for this at https://github.com/wcandillon/xqlint/pull/48

**Comment by RdNetwork** [comment]
Hi,
Any news on this?
Was this ever added to xqlint?

I'm using the [XML Tools for Visual Studio Code](https://github.com/DotJoshJohnson/vscode-xml) extension, which seems to be based on this project, and I run into the same issue.
